### ISSUE TITLE
Wrong __proto__ output?

### ISSUE THREAD

**Issue by jurajkapsz** [lukehaas/RunJS#485]
RunJS `v2.7.1`

The **Output pane** shows by the looks a wrong `__proto__`.

```js
let o1 = { x: 'READ ONLY' };
let o2 = Object.create(o1);

o2; // { __proto__: { x: 'READ ONLY' } }

o2.x = 'NEW';

o2; // { x: 'NEW', __proto__: { x: 'NEW' } } <-- WRONG
o2.__proto__; // { x: 'READ ONLY' } <-- HERE OK
```

![image](https://user-images.githubusercontent.com/2209893/194757184-43e22f34-bc06-42c1-be8f-2f3aa2befa92.png)

**Comment by lukehaas** [comment]
@jurajkapsz thanks for raising this. I'll look into it.

**Comment by lukehaas** [comment]
@jurajkapsz this is now resolved in the latest release - 2.7.4
![Screenshot 2022-12-15 at 11 20 06](https://user-images.githubusercontent.com/741344/207846584-c09b8db9-28ca-4b79-85f3-4c446792805f.png)
